takwimu/views.py

```diff
@@ -152,12 +154,31 @@ def get(self, request, *args, **kwargs):
         search_results = Page.objects.none()
         search_query = request.GET.get('q', '')
         if search_query:
-            search_results = Page.objects.live().search(search_query)
+            topic_results = TopicPage.objects.live().search(
+                search_query)
+            profilepage_results = ProfilePage.objects.live().search(
+                search_query)
+            profilesectionpage_results = ProfileSectionPage.objects.live().search(
+                search_query)
+
```

Comment:
Tested it and works fine @phillipahereza but is it possible to do a single search using `Page` and filtering the results in python, extracting sections from pages and topics from sections, etc? Sort of what we did in the explore data by topic. I feel like making three calls to ElasticSearch every time someone performs a search could end up crippling the system.

PS: Are `TopicPage`s still in use? I thought we're now using `StreamField`s and `TopicBlock`s not `TopicPage`.